addons/l10n_sa_edi/models/account_edi_format.py

```diff
@@ -399,7 +400,11 @@ def _check_move_configuration(self, invoice):
         """
 
         def _set_missing_partner_fields(missing_fields, name):
-            return _("- Please, set the following fields on the %s: %s", name, ', '.join(missing_fields))
+            return _(
+                "- Please, set the following fields on the %(name)s: %(missing_fields)s",
+                name=name,
+                missing_fields=format_list(self.env, missing_fields),
+            )
```

Comment:
This function is only used twice and "name" is not a very good placeholder, I suggest we remove the function and the placeholder `%(name)s`

Reply:
Plus, we could easily imagine this not being translatable in all languages. Something in the sentence might change depending on what "name" contains (e.g. the translation of "the" in French will change depending on the grammatical gender, not to mention German with all its declensions).